homework6/frange.py

```diff
@@ -0,0 +1,35 @@
+class frange:
+    def __init__(self, *args, start=0, end=0, step: float = 1):
```

Comment:
Типізував, але сам передав инт, не чесно :smile: 
```suggestion
    def __init__(self, *args, start=0, end=0, step: float = 1.0):
```